app/Database/CourseQueries.hs

```diff
@@ -9,8 +9,9 @@ and serve the information back to the client.
 -}
 
 module Database.CourseQueries
-    (returnPost,
-     reqsForPost,
+    (retrievePost,
+    returnPost,
```

Comment:
make sure this indentation matches the rest of the names, i.e., aligned with the `retrievePost`, not the `(`